tutorials/physics/using_jolt_physics.rst

```diff
@@ -0,0 +1,272 @@
+for the joint limits, you end up with inverted limits and a potentially strange
+limit shape, especially if your limits allow both linear and angular degrees of
+freedom.
+
+Jolt will behave as if you assigned the body to node_b instead, with node_a
+representing "the world". There is a project setting called :ref:`Physics > Jolt Physics 3D > Joints > World Node<class_ProjectSettings_property_physics/jolt_physics_3d/joints/world_node>`
+that lets you toggle this behavior, if you need compatibility for an existing project.
+
+Collision margins
+~~~~~~~~~~~~~~~~~
+
+Jolt (and other similar physics engines) uses something that Jolt refers to as
+"convex radius" to help improve the performance and behavior of the types of
+collision detection that Jolt relies on for convex shapes. Other physics engines
+(Godot included) might refer to these as "collision margins" instead. Godot exposes
+these as the ``margin`` property on every Shape3D-derived class, but Godot Physics
+itself does not use them for anything.
+
+What these collision margins sometimes do in other engines (as described in Godot's
+documentation) is effectively add a "shell" around the shape, slightly increasing
+its size while also rounding off any edges/corners. In Jolt however, these margins
+are first used to shrink the shape, and then the "shell" is applied, resulting in
+edges/corners being similarly rounded off, but without increasing the size of the
+shape.
+
+To prevent having to tweak this margin property manually, since its default value
+can be problematic for smaller shapes, this module exposes a project setting called
+:ref:`Physics > Jolt Physics 3D > Collisions > Collision Margin Fraction<class_ProjectSettings_property_physics/jolt_physics_3d/collisions/collision_margin_fraction>`
+which is multiplied with the smallest axis of the shape's AABB to calculate the
+actual margin. The margin property of the shape is then instead used as an upper
+bound.
+
+These margins should, for most use-cases, be more or less transparent, but can
+sometimes result in odd collision normals when performing shape queries. You can
+lower the above mentioned project setting to mitigate some of this, including
+setting it to ``0.0``, but too small of a margin can also cause odd collision results,
+so is generally not recommended.
+
+Baumgarte stabilization
+~~~~~~~~~~~~~~~~~~~~~~~
+
+the Baumgarte stabilization Jolt uses can result in some artifacts, like piles of
+bodies not separating as quickly as one might expect.
+
+The strength of this stabilization can be tweaked using the project setting
+:ref:`Physics > Jolt Physics 3D > Simulation > Baumgarte Stabilization Factor<class_ProjectSettings_property_physics/jolt_physics_3d/simulation/baumgarte_stabilization_factor>`.
+Setting this project setting to ``1.0`` will resolve penetration in 1 simulation
+step, resulting in a simulation that more closely resembles Godot Physics, but which
+is also more unstable.
```

Comment:
```suggestion
Setting this project setting to ``0.0`` will turn Baumgarte stabilization off. Setting it to ``1.0`` will resolve penetration in 1 simulation step. This is fast but often also unstable.
```

Reply:
Is it all that useful to mention the `0.0` case? As far as I can tell it just prevents the bodies from moving at all.

Clearly the statement of "more closely resembles Godot Physics" was incorrect on my part, but I must admit I'm a bit confused by this notion that `0.0` would be what disables it, as opposed to `1.0`. I would have thought that a 100% error correction would be the default case in any solver, and that anything less is what constitutes the effect of sacrificing correctness for stability, but perhaps I've got that all wrong.